### PR TITLE
luci-app-aria2: Refactor, new views and more options

### PR DESCRIPTION
**Pull request by kuoruan**
This commit contains:

* Refactor package code.
* Add options for RPC, HTTP/HTTPS and BT.
* Improve descriptions for some options.
* New views for config files and log files.
* Also updated translation for simplified Chinese.

Signed-off-by: Xingwang Liao <kuoruan@gmail.com>

### CODE REVIEW

**Comment by Ansuel** [comment]
@hauke @jow- can this be merged?

**Comment by feckert** [comment]
@kuoruan please rebase against the current master to solve the conflicts

**Comment by kuoruan** [comment]
@feckert Done.

File: applications/luci-app-aria2/Makefile [hunk 1]
```diff
@@ -1,39 +1,18 @@
...
-define Package/$(PKG_NAME)/config
-# shown in make menuconfig <Help>
-help
-		$(LUCI_TITLE)
-		.
-		Version: $(PKG_VERSION)-$(PKG_RELEASE)
-		$(PKG_MAINTAINER)
-endef
+PKG_MAINTAINER:=Xingwang Liao <kuoruan@gmail.com>
+PKG_LICENSE:=MIT
```
Comment (feckert) [inline applications/luci-app-aria2/Makefile]: Why did you changed the PKG_LICENSE to MIT?

Comment (kuoruan) [reply applications/luci-app-aria2/Makefile]: Just because I like it. Most of my open source code is under the MIT license.

Comment (feckert) [reply applications/luci-app-aria2/Makefile]: I don't know if that's so feasible.
What's wrong with the Apache 2.0 license?

@jow- @hnyman Can this be done? From my side I think it is NOK.

Comment (feckert) [reply applications/luci-app-aria2/Makefile]: ping @hnyman any hints from your side

Comment (hnyman) [reply applications/luci-app-aria2/Makefile]: No special hints.   Not an license expert. 

There are MIT licensed packages already in the packages repo, so I do not see anything special in that.  

(The new license would naturally only affect the updated new source code versions.)

I woudl say ACK

**Pull request merged**